config/src/main/java/org/hyperledger/besu/config/EthashConfigOptions.java

```diff
@@ -35,9 +35,14 @@ public OptionalLong getFixedDifficulty() {
     return JsonUtil.getLong(ethashConfigRoot, "fixeddifficulty");
   }
 
+  public OptionalLong getEpochLengthActivationBlock() {
+    return JsonUtil.getLong(ethashConfigRoot, "epochlengthactivation");
+  }
+
   Map<String, Object> asMap() {
     final ImmutableMap.Builder<String, Object> builder = ImmutableMap.builder();
     getFixedDifficulty().ifPresent(l -> builder.put("fixeddifficulty", l));
+    getEpochLengthActivationBlock().ifPresent(a -> builder.put("epochlengthactivation", a));
```

Comment:
I removed `getEpochLengthActivationBlock` from `EthashConfigOptions` because I didn't need it (I created the param in  JsonGenesisConfigOptions instead, and forgot to remove it from here).